internal/provisioner/objects/dataplane/dataplane.go

```diff
@@ -136,14 +136,32 @@ func desiredContainers(contour *model.Contour, contourImage, envoyImage string)
 		ports = append(ports, p)
 	}
 
-	healthPort := 8002
+	var (
+		healthPort  = 8002
```

Comment:
while we're here, it'd be nice to move this constant into `object.go` along with the other port numbers, similar to `EnvoyMetricsPort`.